packages/ui-kit/package.json

```diff
@@ -1,31 +1,42 @@
 {
-  "name": "@propeldata/react-time-series",
-  "version": "0.31.0-rc.6",
+  "name": "@propeldata/ui-kit",
+  "version": "0.1.0",
   "license": "MIT",
   "source": "src/index.ts",
   "main": "dist/index.js",
   "module": "dist/index.module.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "rm -rf dist && tsc",
+    "build": "rm -rf dist && yarn graphql:build && tsc",
+    "graphql:build": "yarn graphql:gen && rm -rf dist && tsc",
+    "graphql:gen": "graphql-codegen --config src/helpers/graphql/codegen.yml && node src/helpers/graphql/script.cjs src/helpers/graphql/generated/index.ts",
     "test": "jest",
     "test:coverage": "jest --ci --coverage --json --outputFile=coverage/coverage.json"
   },
   "publishConfig": {
     "access": "public",
     "group": "@propeldata/ui-kit"
   },
+  "sideEffects": false,
   "dependencies": {
     "@emotion/css": "^11.10.5",
-    "@propeldata/ui-kit-components": "workspace:^0.31.0-rc.4",
-    "@propeldata/ui-kit-graphql": "workspace:^0.31.0-rc.5",
-    "@propeldata/ui-kit-plugins": "workspace:^0.31.0-rc.4",
-    "chart.js": "^4.2.1",
+    "@tanstack/react-query": "^4.29.17",
+    "chart.js": "^4.1.2",
+    "chartjs-adapter-date-fns": "2.0.1",
     "date-fns": "^2.29.3",
-    "graphql": "^16.6.0"
+    "dotenv": "^16.0.3",
+    "graphql": "^16.6.0",
```

Comment:
graphql is also in peer dependencies. If it's in peer dependencies, should this actually move to dev dependencies?

Reply:
Here `graphql` is also in `dependencies`, AFAIK this is useful in case the consuming application does not use `graphql` then the ui-kit will install its own but if the application does have `graphql` then the ui-kit will use that one.

Reply:
Yeah,  dev deps seems like a better place. I'll try that.